premake5.lua

```diff
@@ -903,7 +903,10 @@ tool "slang-embed"
 tool "slang-test"
     uuid "0C768A18-1D25-4000-9F37-DA5FE99E3B64"
     includedirs { "." }
-    links { "compiler-core", "slang", "core", "miniz", "lz4" }
+    -- We link with llvm here for FileCheck support, LLVM-14 is the
+    -- library required according to llvm-config (llvm-config isn't in the
+    -- binary llvm distribution on Windows, so just inline the output here)
+    links { "compiler-core", "slang", "core", "miniz", "lz4", "LLVM-13" }
```

Comment:
This is not done through `slang-llvm` wrapper?
Does this mean I need to pull llvm repo locally and setup my environment so the llvm headers can be found?